packages/beacon-node/src/network/nodejs/bundle.ts

```diff
@@ -35,6 +36,9 @@ export async function createNodejsLibp2p(options: ILibp2pOptions): Promise<Libp2
       peerDiscovery.push(new MulticastDNS());
     }
   }
+
+  const yamuxInstance = yamux()();
```

Comment:
Note to self: do not merge this change, yamux must be tested and merged on its own